server/frontend/package.json

```diff
@@ -14,13 +14,16 @@
     },
     "dependencies": {
         "axios": "^1.6.7",
+        "chart.js": "^4.4.7",
```

Comment:
Is chartJS still needed after this change?

Reply:
We noticed some other references to chartjs that we weren’t entirely sure about, so we decided it was safer to leave it for now. However, I’ll review its usage and remove it if it’s no longer required.

Reply:
Chartjs is used in ec2spotmanager, but we no longer use ec2spotmanager anymore at all.